tests/step-117.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+# Test:
+#   Test urls and local paths in shared hooks
+
+if ! sh /var/lib/githooks/install.sh; then
+    echo "! Failed to execute the install script"
+    exit 1
+fi
+
+# Make repo
+mkdir -p /tmp/test117 && cd /tmp/test117 || exit 1
+git init &&
+    mkdir ".githooks" &&
+    touch ".githooks/trust-all" &&
+    git add . &&
+    git commit -a -m 'Initial commit' ||
+    exit 1
+
+# Simulate a shared hook repo on the server
+mkdir -p /tmp/shared/shared-server.git &&
+    cd /tmp/shared/shared-server.git &&
+    git init --bare &&
+    git clone /tmp/shared/shared-server.git /tmp/shared/shared.git &&
+    cd /tmp/shared/shared.git &&
+    mkdir -p .githooks/pre-commit &&
+    git config core.hooksPath "/dev/null" && # dont execute hooks in this repo!
```

Comment:
Hm, we shouldn't execute hooks in our shared repos anyway I think?
Doesn't the hook executor pass in some magic git parameters to the `git pull` to not run hooks?

Reply:
Or is this just to avoid triggering on the `git commit` and `git push` here in the test below?

Reply:
Correct. I actually found some Bugs related to this! -> I addressed in c369be70 exactly this problems.